Dockerfile

```diff
@@ -18,9 +18,7 @@ COPY --from=builder /app .
 RUN pip install --upgrade pip && \
     pip install -r requirements.txt
 
-RUN python manage.py migrate
-
 EXPOSE 8080
 
 # Run database migrations and start the Django application
-ENTRYPOINT ["python", "manage.py", "runserver", "0.0.0.0:8080"]
+ENTRYPOINT ["sh", "-c", "python manage.py migrate && python manage.py runserver 0.0.0.0:8080"]
```

Comment:
![image](https://github.com/mate-academy/devops_todolist_docker_core_task_3_docker_compose/assets/87720583/938397cb-d515-4cd8-9e9d-55fe271fd3dc)
Add Eof(End-of-file), need to add empty line at the end

Reply:
fixed